pcsx2/gui/Panels/VideoPanel.cpp

```diff
@@ -30,7 +30,7 @@ Panels::FramelimiterPanel::FramelimiterPanel( wxWindow* parent )
 	: BaseApplicableConfigPanel_SpecificConfig( parent )
 {
 	m_check_LimiterDisable = new pxCheckBox( this, _("Disable Framelimiting"),
-		_("Useful for running benchmarks. Toggle this option in-game by pressing F4.") );
+		_("Uncaps FPS. Toggle in-game by pressing F4. (Useful for running benchmarks.)" ) );
```

Comment:
Brackets seem odd here, maybe remove them.
`_("Uncaps FPS. Toggle in-game by pressing F4. Useful for running benchmarks." ) );`